sdk/go/common/resource/plugin/plugin.go

```diff
@@ -183,6 +184,15 @@ func newPlugin(ctx *Context, pwd, bin, prefix string, kind workspace.PluginKind,
 		logging.V(9).Infof("Launching plugin '%v' from '%v' with args: %v", prefix, bin, argstr)
 	}
 
+	// Create a root span for the operation
+	opts := []opentracing.StartSpanOption{}
+	if ctx != nil && ctx.tracingSpan != nil {
+		opts = append(opts, opentracing.ChildOf(ctx.tracingSpan.Context()))
+	}
+	opts = append(opts, opentracing.Tag{Key: "pulumi-decorator", Value: prefix + ":" + bin})
+	tracingSpan := opentracing.StartSpan("LaunchPlugin", opts...)
```

Comment:
I'm not totally up to best practices with tracing, but I'd think this would be easier to match up if it matched the method name "newPlugin"?

Reply:
We're a bit ad-hoc there. I can go either way.

Reply:
I was trying to follow the log line a couple lines above, perhaps even better would be to make both the span and the log line reflect the method name.